mcp/ServerInfo.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Wachterjohannes\Mcp;
+
+use Mcp\Capability\Attribute\McpTool;
+
+class ServerInfo
```

Comment:
I thought it would be okey if we added some basic MCP tools that are not coupled to any framework.